Umbraco-Cloud/Troubleshooting/index.md

```diff
@@ -12,4 +12,171 @@ To help you troubleshoot you should start by determining where this error is rep
 
 ## Is your issue related to something else?
 - [Minor upgrades issues](../Upgrades/Minor-Upgrades/)
-- [Baseline merge conflicts](../Getting-Started/Baselines/Baseline-Merge-Conflicts)
\ No newline at end of file
+- [Baseline merge conflicts](../Getting-Started/Baselines/Baseline-Merge-Conflicts)
```

Comment:
Don't know if you had a bad rebase or something, but this whole page has been modified greatly so it no longer looks like this, all of the information below has been moved / deleted / rewritten and should not be on this page anymore.